Vagrantfile

```diff
@@ -3,12 +3,11 @@
 
 Vagrant.configure("2") do |config|
     # Start with base vagrant box. This one is only 271 MB in size.
-    config.vm.box = "ubuntu/trusty64"
+    config.vm.box = "ubuntu/xenial64"
```

Comment:
Maybe for next summit? I don't think we would get time to test for the summit today with this.